src/tests/fixtures/unobtrusive_javascript.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Unobtrustive JavaScript</title>
+    <script type="module">
+      import "/dist/turbo.es2017-umd.js"
```

Comment:
This is currently limited to a Functional test HTML fixture file. I struggled to declare the code within this `<script>` tag in an ESM/UMD module, and then optionally import it. Unfortunately, I was unable to figure it out.

If anyone could provide guidance, I'd prefer to declare something like:

```js
export function startUjs(Rails: any) {
  const { delegate, disableElement, enableElement } = Rails

  delegate(document, Rails.linkDisableSelector,   "turbo:before-cache", enableElement)
  delegate(document, Rails.buttonDisableSelector, "turbo:before-cache", enableElement)
  delegate(document, Rails.buttonDisableSelector, "turbo:submit-end", enableElement)

  delegate(document, Rails.formSubmitSelector, "turbo:submit-start", disableElement)
  delegate(document, Rails.formSubmitSelector, "turbo:submit-end", enableElement)
  delegate(document, Rails.formSubmitSelector, "turbo:before-cache", enableElement)
}

// ...
import Rails from "..."
import { startUjs } from "@hotwire/turbo/ujs"

startUjs(Rails)
```